lib/main.dart

```diff
@@ -16,12 +23,58 @@ class MyApp extends StatelessWidget {
         theme: ThemeData(
           primarySwatch: Colors.teal,
         ),
-        home: Scaffold(
-            appBar: AppBar(
-              title: const Text('DWYL'),
-              centerTitle: true,
-            ),
-            body: const MyTextField()));
+        home: const HomePage());
+  }
+}
+
+class HomePage extends StatefulWidget {
+  const HomePage({super.key});
+
+  @override
+  State<HomePage> createState() => _HomePageState();
+}
+
+class _HomePageState extends State<HomePage> {
+  bool showMenu = true;
+  final GlobalKey<ScaffoldState> _scaffoldKey = GlobalKey<ScaffoldState>();
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        key: _scaffoldKey,
+        drawerEnableOpenDragGesture: false,
+        appBar: AppBar(
+          key: appBarKey,
+          title: Row(
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: [
+              Image.asset("assets/images/dwyl_logo.png", fit: BoxFit.fitHeight, height: 30),
```

Comment:
Is there a reason for including a `new` instance of the dwyl logo when the previous PR added so many of them?
e.g: https://github.com/dwyl/app/blob/main/assets/icon/icon.png 🤷‍♂️

Reply:
It's because this PR was created prior to the previous one being merged. There wasn't any images back then.
You can assign the PR back to me so I can merge `main` to this PR.

Reply:
Please try to re-use the logo image if possible. 🤞

Reply:
Should be fixed by [`44acd24` (#314)](https://github.com/dwyl/app/pull/314/commits/44acd2452a9818f705e521ea7e6a9f0839ee58b0).